mycroft/skills/phillips_hue/vocab/en-us/ActivateSceneKeyword.voc

```diff
@@ -0,0 +1,6 @@
+activate scene
+activate seen
+set my lights to
+set scene
+set seen
```

Comment:
I agree, making sure the user knows what is going on is very important, One of the big reasons why I don't want to add the extra vocab is to prevent future unnecessary vocab conflicts. 

From what I've seen from the logs, the STT gets multiple interpretations of the given utterance, and then picks the one that it has the highest confidence level for, presumably based on grammatical correctness and some other characteristics. That's why I would try to say something like `activate the scene reading`, where `activate the seen reading` would have a lower confidence level. It would probably be better to test that though, as I don't actually know how significant that difference would be.